config/locales/en.yml

```diff
@@ -95,12 +95,21 @@ en:
     logout: logout
     permissions: "Permissions"
     permissions_heading: "Your Permissions"
+    change_password_heading: "Change Your Password"
     access: "Access"
     client_switch_heading: "Switch Authentication Clients"
     client_switch_legend: "In development you can switch between using cognito and the stub authentication clients. If you use the stub client you can assume different roles."
     banner: "There are features on this page which are only avialable in the development environment."
     assume_roles_heading: "Switch Roles"
     assume_roles_legend: "Select which role(s) you wish to assume?"
+    old_password_mismatch: "Your old password does not match the one we have on record"
```

Comment:
could the errors be shifted to `errors` section (still within profile, but group errors together)